src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -3085,6 +3088,28 @@ private void TabItem_MouseLeftButtonUp(object sender, MouseButtonEventArgs e)
                 this.dynamoViewModel.ShowStartPage = false;
         }
 
+        private void PinHomeButton()
+        {
+            const int FirstTabItemMinimumLeftMarginOffset = 230;
```

Comment:
These constants are defined in UpdateWorkspaceTabSizes also, can you put them in the class and remove duplicates.

Reply:
I was hesitant to touch the DynamoView class further, but thought about it too. That's now done, but I opted to keep the class variables close to the first place where they are used to reduce clutter.